### PR TITLE
Fix sorting array with mixed BigInt and Number values

### PR DESCRIPTION
**Pull request by timagixe**
`TypeError: Cannot convert a BigInt value to a number` error message appears when sorting mixed array (both with BigInt and Number values) with custom sort function (second example):

```
console.log(array1.sort((a, b) => BigInt(a) - BigInt(b)));
                   ^

TypeError: Cannot convert a BigInt value to a number
    at Array.sort (<anonymous>)
```

`sort()` expects to receive a `compareFn` which returns `1`, `-1` or `0`. This expression `BigInt(a) - BigInt(b)` was returning result of BigInt type which led to a TypeError.

I have also actualized an error message in a commented code.

### CODE REVIEW

**Comment by timagixe** [comment]
For reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#comparisons

File: JavaScript/9-bigint.js [hunk 1]
```diff
@@ -51,7 +51,13 @@ console.log();
...
 console.log(array1);
 console.log(array1.sort());
-console.log(array1.sort((a, b) => BigInt(a) - BigInt(b)));
+console.log(array1.sort((a, b) => {
+  const biA = BigInt(a);
+  const biB = BigInt(b);
+  if (biA < biB) return -1;
+  if (biA > biB) return 1;
+  return 0;
+}));
```
Comment (tshemsedinov) [inline JavaScript/9-bigint.js:60]:
```suggestion
console.log(array1.sort((a, b) => Number(BigInt(a) - BigInt(b)));
```

**Review by tshemsedinov — COMMENTED** [review]
We can just cast bigint back to the number

File: JavaScript/9-bigint.js (hunk 1 above)
Comment (timagixe) [reply JavaScript/9-bigint.js]: I also thought of this, but it won't work when there will really huge BigInt values in the array.

Changed in [86a18cf](https://github.com/HowProgrammingWorks/DataTypes/pull/137/commits/86a18cf974d1a816a056fc261fb2c8179e70937f)

**Review by tshemsedinov — APPROVED** [review]

**Pull request merged**